File: src/main/java/net/atos/zac/mail/MailjetClientHelper.java
package net.atos.zac.mail;

import com.mailjet.client.ClientOptions;
import com.mailjet.client.MailjetClient;

public class MailjetClientHelper {
    public static MailjetClient createMailjetClient(String mailjetApiKey, String mailjetSecretKey) {
        return new MailjetClient(
                                 ClientOptions.builder()
                                              .apiKey(mailjetApiKey)
                                              .apiSecretKey(mailjetSecretKey)
                                              .build()
        );
    }
}

File: src/main/kotlin/net/atos/zac/mail/MailjetClientHelper.kt
package net.atos.zac.mail

import com.mailjet.client.ClientOptions
import com.mailjet.client.MailjetClient

object MailjetClientHelper {
    fun createMailjetClient(mailjetApiKey: String?, mailjetSecretKey: String?): MailjetClient {
        return MailjetClient(
            ClientOptions.builder()
                .apiKey(mailjetApiKey)
                .apiSecretKey(mailjetSecretKey)
                .build()
        )
    }
}
